cypress/page/download.page.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable require-jsdoc */
+
+class DownloadFilePage {
+  private pageURL: string;
+  private downloadTxtFile: string;
+  private generateTxtFile: string;
+  private TxtDownloadButton: string;
+
+  private downloadPdfFile   : string;
+  private generatePdfFile: string;
+  private TxtDownPdfButton: string;
+
+  constructor() {
+    this.pageURL = "http://demo.automationtesting.in/FileDownload.html";
+    this.downloadTxtFile = "#textbox";
+    this.generateTxtFile = "#createTxt";
+    this.TxtDownloadButton = "#link-to-download";
+    this.downloadPdfFile     = "#pdfbox";
```

Comment:
can you delete the whitespaces between `downloadPdfFile` and `=` leaving just one